Sources/Sentry/SentryViewHierarchyIntegration.m

```diff
@@ -13,7 +13,8 @@
 void
 saveViewHierarchy(const char *path)
```

Comment:
- [ ] `m`: change `path` to something more descriptive like `reportDirectory`
- [ ] `l`: add headerdoc above the signature. it's obvious here now, but if it gets refactored away may not be
```suggestion
/**
* Function to call through to the ObjC method to save a view hierarchy, which can be passed around as
* a function pointer in the C crash reporting code.
* @param reportDirectoryPath The path to the directory containing crash reporting files, in which a new
* file will be created to store the view hierarchy description.
*/
void
saveViewHierarchy(const char *reportDirectoryPath)
```

Reply:
I think we should also update `saveScreenShot` with a similar comment.